CRITICAL - 2021-08-26 00:05:42 --> Array to string conversion
#0 C:\laragon\www\v1\app\Entities\User.php(16): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\laragon\\www\\...', 16, Array)
#1 C:\laragon\www\v1\app\Controllers\Auth\login.php(20): App\Entities\User->generateUsarname()
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2021-08-26 14:29:19 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-08-26 14:29:19 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\BaseModel->save(Array)
#7 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2021-08-26 14:30:03 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-26 14:30:03 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#11 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2021-08-26 14:32:18 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-26 14:32:18 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#11 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2021-08-26 14:32:19 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-26 14:32:19 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#11 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2021-08-26 14:33:57 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-26 14:33:57 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 14:36:12 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-26 14:36:12 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 14:38:47 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-26 14:38:47 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 14:38:49 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-26 14:38:49 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 14:39:00 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-26 14:39:00 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 14:45:32 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-26 14:45:32 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 14:45:50 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-26 14:45:50 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_users_info` FOREIGN KEY (`info_id_user`) REFERENCES `users_info` (`id_user`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 15:33:03 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_groups1` FOREIGN KEY (`groups_id_group`) REFERENCES `groups` (`id_group`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-26 15:33:03 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users`, CONSTRAINT `fk_users_groups1` FOREIGN KEY (`groups_id_group`) REFERENCES `groups` (`id_group`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 15:35:49 --> Duplicate entry '' for key 'username_UNIQUE'
CRITICAL - 2021-08-26 15:35:49 --> Duplicate entry '' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2021-08-26 15:39:22 --> Call to protected method App\Entities\User::generateUsarname() from context 'App\Controllers\Auth\login'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
ERROR - 2021-08-26 15:40:10 --> Duplicate entry '' for key 'username_UNIQUE'
CRITICAL - 2021-08-26 15:40:10 --> Duplicate entry '' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(21): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 15:40:36 --> Duplicate entry '' for key 'username_UNIQUE'
CRITICAL - 2021-08-26 15:40:36 --> Duplicate entry '' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(21): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 15:55:41 --> Duplicate entry '' for key 'username_UNIQUE'
CRITICAL - 2021-08-26 15:55:41 --> Duplicate entry '' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 15:55:43 --> Duplicate entry '' for key 'username_UNIQUE'
CRITICAL - 2021-08-26 15:55:43 --> Duplicate entry '' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(22): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 15:56:11 --> Duplicate entry 'alejandro_fvega@gmail.com' for key 'email_UNIQUE'
CRITICAL - 2021-08-26 15:56:11 --> Duplicate entry 'alejandro_fvega@gmail.com' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(24): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 22:55:52 --> Duplicate entry '' for key 'username_UNIQUE'
CRITICAL - 2021-08-26 22:55:52 --> Duplicate entry '' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(687): CodeIgniter\Model->insert(Object(App\Entities\User), false)
#8 C:\laragon\www\v1\app\Controllers\Auth\login.php(23): CodeIgniter\BaseModel->save(Object(App\Entities\User))
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#11 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2021-08-26 22:56:32 --> Duplicate entry '' for key 'username_UNIQUE'
CRITICAL - 2021-08-26 22:56:32 --> Duplicate entry '' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(687): CodeIgniter\Model->insert(Object(App\Entities\User), false)
#8 C:\laragon\www\v1\app\Controllers\Auth\login.php(23): CodeIgniter\BaseModel->save(Object(App\Entities\User))
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#11 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2021-08-26 22:56:34 --> Duplicate entry '' for key 'username_UNIQUE'
CRITICAL - 2021-08-26 22:56:34 --> Duplicate entry '' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(687): CodeIgniter\Model->insert(Object(App\Entities\User), false)
#8 C:\laragon\www\v1\app\Controllers\Auth\login.php(23): CodeIgniter\BaseModel->save(Object(App\Entities\User))
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#11 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2021-08-26 22:58:40 --> Duplicate entry '' for key 'username_UNIQUE'
CRITICAL - 2021-08-26 22:58:40 --> Duplicate entry '' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(687): CodeIgniter\Model->insert(Object(App\Entities\User), false)
#8 C:\laragon\www\v1\app\Controllers\Auth\login.php(23): CodeIgniter\BaseModel->save(Object(App\Entities\User))
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#11 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2021-08-26 22:59:39 --> Duplicate entry '' for key 'username_UNIQUE'
CRITICAL - 2021-08-26 22:59:39 --> Duplicate entry '' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(687): CodeIgniter\Model->insert(Object(App\Entities\User), false)
#8 C:\laragon\www\v1\app\Controllers\Auth\login.php(23): CodeIgniter\BaseModel->save(Object(App\Entities\User))
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#11 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2021-08-26 22:59:50 --> Duplicate entry '' for key 'username_UNIQUE'
CRITICAL - 2021-08-26 22:59:50 --> Duplicate entry '' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(687): CodeIgniter\Model->insert(Object(App\Entities\User), false)
#8 C:\laragon\www\v1\app\Controllers\Auth\login.php(23): CodeIgniter\BaseModel->save(Object(App\Entities\User))
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#11 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2021-08-26 23:00:21 --> Duplicate entry 'alejadro_fveg@gmail.com' for key 'email_UNIQUE'
CRITICAL - 2021-08-26 23:00:21 --> Duplicate entry 'alejadro_fveg@gmail.com' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(687): CodeIgniter\Model->insert(Object(App\Entities\User), false)
#8 C:\laragon\www\v1\app\Controllers\Auth\login.php(23): CodeIgniter\BaseModel->save(Object(App\Entities\User))
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#11 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2021-08-26 23:04:36 --> Duplicate entry '' for key 'username_UNIQUE'
CRITICAL - 2021-08-26 23:04:36 --> Duplicate entry '' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(23): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 23:04:57 --> Duplicate entry 'alejadro_fveggmail.com' for key 'email_UNIQUE'
CRITICAL - 2021-08-26 23:04:57 --> Duplicate entry 'alejadro_fveggmail.com' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(23): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 23:06:37 --> Duplicate entry 'alejadro_fveggmail.com' for key 'email_UNIQUE'
CRITICAL - 2021-08-26 23:06:37 --> Duplicate entry 'alejadro_fveggmail.com' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(23): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-26 23:28:27 --> Duplicate entry 'alejadro_fve@ggmail.com' for key 'email_UNIQUE'
CRITICAL - 2021-08-26 23:28:27 --> Duplicate entry 'alejadro_fve@ggmail.com' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(32): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2021-08-26 23:44:26 --> Argument 1 passed to App\Models\UserModel::withGroup() must be of the type string, object given, called in C:\laragon\www\v1\app\Controllers\Auth\login.php on line 33
#0 C:\laragon\www\v1\app\Controllers\Auth\login.php(33): App\Models\UserModel->withGroup(Object(Config\Configuration))
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
